source/_integrations/cover.mysensors.markdown

```diff
@@ -149,3 +149,9 @@ void receive(const MyMessage &message) {
   }
 }
 ```
+
+## Sketch example with position measurement based on motor running time
+
+This sketch is ideally for star topology wiring. You can run up to 12 covers with a single Arduino Mega board and some releys. All you need to set is one line of parameters for one Cover. However you can also use it for a single cover based on an arduino nano or even ESP8266 board.
+
+https://github.com/gryzli133/RollerShutterSplit
```

Comment:
This is a bare link, please don't use those.

Reply:
the sketch is built of multiple files, so I cannot easily paste the code...

Reply:
That is fine, but please don't use bare links.

Reply:
I changed that. Should be OK now.